conans/client/generators/cmake.py

```diff
@@ -21,7 +22,8 @@ def __init__(self, deps_cpp_info):
         self.exelinkflags = " ".join(deps_cpp_info.exelinkflags)
         self.bin_paths = "\n\t\t\t".join('"%s"' % p.replace("\\", "/")
                                          for p in deps_cpp_info.bin_paths)
-
+        self.build_paths = "\n\t\t\t".join('"%s"' % p.replace("\\", "/")
```

Comment:
Maybe an internal function for ""\n\t\t\t".join('"%s"' % p.replace("\\", "/").... etc"? too repeated